src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -31,16 +31,16 @@ import {CHANNEL_GROUP, GROUP_INFO, SIGNED} from '../../hooks/core/constant';
 import {Context} from '../../context';
 import {Loading} from '../../components';
 import {ProfileContact} from '../../components/Items';
-import {colors} from '../../utils/colors';
 import {fonts, normalize, normalizeFontSize} from '../../utils/fonts';
 import {getChatName} from '../../utils/string/StringUtils';
 import {isContainUrl} from '../../utils/Utils';
+import {COLORS} from '../../utils/theme';
 import MemoIc_pencil from '../../assets/icons/Ic_pencil';
 import {channelImageStyles} from '../../components/ChatList/elements/ChannelImage.style';
 import useProfileHook from '../../hooks/core/profile/useProfileHook';
 
 export const styles = StyleSheet.create({
-  container: {flex: 1, backgroundColor: '#fff', paddingBottom: 40},
+  container: {flex: 1, backgroundColor: COLORS.white, paddingBottom: 40},
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `colors` module to `COLORS` object from the `theme` module is a good move towards better maintainability and consistency. However, it's important to ensure that all instances of the old `colors` module are replaced throughout the application to avoid any inconsistencies.
```diff
- import {colors} from '../../utils/colors';
+ import {COLORS} from '../../utils/theme';
...
- container: {flex: 1, backgroundColor: '#fff', paddingBottom: 40},
+ container: {flex: 1, backgroundColor: COLORS.white, paddingBottom: 40},
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->